docs/experimental-features/host-risk-score.asciidoc

```diff
@@ -23,12 +25,15 @@ The following table shows how risk levels are applied to a host, based on the no
 |==============================================
 
 [discrete]
-=== Deploy the Host Risk Score package
+=== Deploy the host risk score package
+
+To deploy the host risk score framework in your environment, follow https://github.com/elastic/detection-rules/blob/main/docs/experimental-machine-learning/host-risk-score.md[these steps]. These instructions also include steps to enable the `riskyHostsEnabled` feature flag.
 
-To deploy the Host Risk Score framework in your environment, follow https://github.com/elastic/detection-rules/blob/main/docs/experimental-machine-learning/host-risk-score.md[these steps]. These instructions also include steps to enable the `riskyHostsEnabled` feature flag.
+NOTE: Host risk score artifacts should be updated when you upgrade the {stack}. To do this, download a release bundle that's compatible with the new {stack} version and repeat all the steps referenced above. Failure to do so might cause views in the {security-app} to break.
 
-NOTE: To view host risk score data on the {elastic-sec} Overview page and the Alert details flyout, you must enable the feature flag. However, enabling the feature flag is NOT required to view the Lens dashboards.
+NOTE: To view host risk score data in the {security-app}, you must enable the feature flag. However, enabling the feature flag is NOT required to view the Lens dashboards.
```

Comment:
What do you think about linking to [this set of instructions](https://github.com/elastic/detection-rules/blob/main/docs/experimental-machine-learning/host-risk-score.md#8-enable-kibana-features) in this note. That way, we guide users towards the steps to enable the feature flag. 

Also, would it be better to define the feature flag here (`riskyHostsEnabled`) instead of first defining it in the following section?